sequencer/batchbuilder.go

```diff
@@ -171,11 +205,11 @@ func (s *Sequencer) isSequenceProfitable(ctx context.Context) bool {
 }
 
 func (s *Sequencer) processTxs(ctx context.Context, pendingTxs []*pool.Transaction) (
-	[]*state.ProcessTransactionResponse, []common.Hash, map[string]*state.ProcessTransactionResponse, error) {
+	[]*state.ProcessTransactionResponse, []common.Hash, map[string]*state.ProcessTransactionResponse, bool, error) {
```

Comment:
Amount of variables to return has become too large. I think, it will be better to return struct here
```
type processTxsResponse struct {
  processedTxs []*state.ProcessTransactionResponse
  processedTxsHashes []common.Hash
  unprocessedTxs map[string]*state.ProcessTransactionResponse
  isBatchNotProcessed bool
}  
```